src/App.jsx

```diff
@@ -14,70 +16,100 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const START_SELECTED_GOOD = 'Jam';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(START_SELECTED_GOOD);
 
+  const selectGood = (name) => {
+    const callback = () => setSelectedGood(name);
```

Comment:
There is an unnecessary callback. Anyway, you are using a second method.

Reply:
agree